src/service_daemon.rs

```diff
@@ -649,6 +647,19 @@ impl ServiceDaemon {
                 }
             },
 
+            Command::Resolve(instance, try_count) => {
+                zc.query_missing_srv(&instance);
+                if try_count < 1 {
+                    // Only repeat 1 time at most. Note that if the curret try already
```

Comment:
This is nice, nitty gritty: 'current' :)

Reply:
Thanks! Fixed.